install/kubernetes/istio-rbac-alpha.yaml

```diff
@@ -1,102 +1,16 @@
 # Permissions and roles for istio
-# To debug: start the cluster with -vmodule=rbac,3 to enable verbose logging on RBAC DENY
-# Also helps to enable logging on apiserver 'wrap' to see the URLs.
-# Each RBAC deny needs to be mapped into a rule for the role.
-# If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
-kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
```

Comment:
I made a change to fix namespaces - replace ClusterRoleBinding with RoleBinding, and remove the namespeca:default.

The rest seems reasonable to unblock alpha.